<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0">
      <property name="margin">
       <number>1</number>
      </property>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Propriétés</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_7">
             <item row="0" column="2">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Sous catégorie</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Categorie</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="comboBox_cat">
               <property name="enabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QComboBox" name="comboBox_souscat"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QStackedWidget" name="stackedWidget">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="page">
             <layout class="QGridLayout" name="gridLayout">
              <property name="margin">
               <number>3</number>
              </property>
              <item row="0" column="1">
               <widget class="QDateEdit" name="dateEdit_miseenservice"/>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Date de mise en service</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0" colspan="2">
               <widget class="QStackedWidget" name="stackedWidget_2">
                <property name="currentIndex">
                 <number>0</number>
                </property>
                <widget class="QWidget" name="page_3">
                 <layout class="QGridLayout" name="gridLayout_2">
                  <property name="horizontalSpacing">
                   <number>3</number>
                  </property>
                  <property name="margin">
                   <number>3</number>
                  </property>
                  <item row="1" column="0">
                   <widget class="QLabel" name="label_5">
                    <property name="text">
                     <string>Numéro de série</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="0">
                   <widget class="QLabel" name="label">
                    <property name="text">
                     <string>Durée de vie</string>
                    </property>
                   </widget>
                  </item>
                  <item row="4" column="1">
                   <widget class="QSpinBox" name="spinBox_freqpannes"/>
                  </item>
                  <item row="2" column="0">
                   <widget class="QLabel" name="label_6">
                    <property name="text">
                     <string>Exposition aux conditions climatiques</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="1">
                   <widget class="QComboBox" name="comboBox_expoconditionclim"/>
                  </item>
                  <item row="4" column="0">
                   <widget class="QLabel" name="label_8">
                    <property name="text">
                     <string>Fréquence panne (mois)</string>
                    </property>
                   </widget>
                  </item>
                  <item row="4" column="2">
                   <widget class="QLabel" name="label_10">
                    <property name="text">
                     <string>Fréquence renouvellement (mois)</string>
                    </property>
                   </widget>
                  </item>
                  <item row="4" column="3">
                   <widget class="QSpinBox" name="spinBox_freqrenouv"/>
                  </item>
                  <item row="2" column="2">
                   <widget class="QLabel" name="label_7">
                    <property name="text">
                     <string>Fréquence maintenance (mois)</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="3">
                   <widget class="QSpinBox" name="spinBox_freqmaintenance"/>
                  </item>
                  <item row="0" column="1" colspan="3">
                   <widget class="QSpinBox" name="spinBox_dureevie"/>
                  </item>
                  <item row="1" column="1" colspan="3">
                   <widget class="QLineEdit" name="lineEdit_numserie"/>
                  </item>
                  <item row="6" column="0" colspan="4">
                   <widget class="QStackedWidget" name="stackedWidget_3">
                    <property name="currentIndex">
                     <number>2</number>
                    </property>
                    <widget class="QWidget" name="page_5">
                     <layout class="QGridLayout" name="gridLayout_3">
                      <property name="margin">
                       <number>3</number>
                      </property>
                      <item row="0" column="1">
                       <widget class="QComboBox" name="comboBox_lisibilite"/>
                      </item>
                      <item row="0" column="0">
                       <widget class="QLabel" name="label_12">
                        <property name="text">
                         <string>Lisibilité information</string>
                        </property>
                       </widget>
                      </item>
                      <item row="1" column="0" colspan="2">
                       <spacer name="verticalSpacer_2">
                        <property name="orientation">
                         <enum>Qt::Vertical</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                         <size>
                          <width>20</width>
                          <height>40</height>
                         </size>
                        </property>
                       </spacer>
                      </item>
                     </layout>
                    </widget>
                    <widget class="QWidget" name="page_6">
                     <layout class="QGridLayout" name="gridLayout_4">
                      <property name="margin">
                       <number>3</number>
                      </property>
                      <item row="1" column="1">
                       <widget class="QComboBox" name="comboBox_adaptbruits"/>
                      </item>
                      <item row="1" column="0">
                       <widget class="QLabel" name="label_14">
                        <property name="text">
                         <string>Adaptation au bruit environnant</string>
                        </property>
                       </widget>
                      </item>
                      <item row="0" column="1">
                       <widget class="QComboBox" name="comboBox_couvsonore"/>
                      </item>
                      <item row="0" column="0">
                       <widget class="QLabel" name="label_13">
                        <property name="text">
                         <string>Couverture sonore</string>
                        </property>
                       </widget>
                      </item>
                      <item row="3" column="0" colspan="2">
                       <spacer name="verticalSpacer_3">
                        <property name="orientation">
                         <enum>Qt::Vertical</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                         <size>
                          <width>20</width>
                          <height>40</height>
                         </size>
                        </property>
                       </spacer>
                      </item>
                      <item row="1" column="2">
                       <widget class="QLabel" name="label_15">
                        <property name="text">
                         <string>Audibilité des informations</string>
                        </property>
                       </widget>
                      </item>
                      <item row="1" column="3">
                       <widget class="QComboBox" name="comboBox_audibinfo"/>
                      </item>
                     </layout>
                    </widget>
                    <widget class="QWidget" name="page_7">
                     <layout class="QGridLayout" name="gridLayout_5">
                      <property name="horizontalSpacing">
                       <number>3</number>
                      </property>
                      <property name="margin">
                       <number>3</number>
                      </property>
                      <item row="1" column="2">
                       <widget class="QLabel" name="label_18">
                        <property name="text">
                         <string>Technologie utilisée</string>
                        </property>
                       </widget>
                      </item>
                      <item row="1" column="1">
                       <widget class="QComboBox" name="comboBox_typecamera"/>
                      </item>
                      <item row="1" column="0">
                       <widget class="QLabel" name="label_16">
                        <property name="text">
                         <string>Type de caméra</string>
                        </property>
                       </widget>
                      </item>
                      <item row="3" column="1">
                       <widget class="QComboBox" name="comboBox_couvvideo"/>
                      </item>
                      <item row="3" column="0">
                       <widget class="QLabel" name="label_19">
                        <property name="text">
                         <string>Couverture vidéo de la zone</string>
                        </property>
                       </widget>
                      </item>
                      <item row="1" column="3">
                       <widget class="QComboBox" name="comboBox_technocamera"/>
                      </item>
                      <item row="3" column="2">
                       <widget class="QLabel" name="label_20">
                        <property name="text">
                         <string>Note clarté des images /10</string>
                        </property>
                       </widget>
                      </item>
                      <item row="3" column="3">
                       <widget class="QSpinBox" name="spinBox_clarteimages"/>
                      </item>
                      <item row="0" column="0">
                       <widget class="QLabel" name="label_17">
                        <property name="text">
                         <string>Type de zone de manœuvre couverte</string>
                        </property>
                       </widget>
                      </item>
                      <item row="0" column="1" colspan="3">
                       <widget class="QLineEdit" name="lineEdit_zonemanoeuvre"/>
                      </item>
                     </layout>
                    </widget>
                   </widget>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="page_4">
                 <layout class="QGridLayout" name="gridLayout_6">
                  <property name="topMargin">
                   <number>3</number>
                  </property>
                  <property name="rightMargin">
                   <number>3</number>
                  </property>
                  <property name="bottomMargin">
                   <number>3</number>
                  </property>
                  <property name="spacing">
                   <number>3</number>
                  </property>
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_21">
                    <property name="text">
                     <string>Type de maintenance</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLineEdit" name="lineEdit_typemaintenance"/>
                  </item>
                  <item row="1" column="0" colspan="2">
                   <spacer name="verticalSpacer">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>40</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_2">
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <property name="margin">
               <number>3</number>
              </property>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>cat2</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_3">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_8">
             <property name="margin">
              <number>3</number>
             </property>
             <property name="spacing">
              <number>3</number>
             </property>
             <item row="0" column="1">
              <widget class="QTextBrowser" name="textBrowser_comm">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>60</height>
                </size>
               </property>
               <property name="readOnly">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Commentaire</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Désordres</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QFrame" name="frame_desordres">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout"/>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
